app/views/notification.js

```diff
@@ -0,0 +1,31 @@
+import Mn from 'backbone.marionette';
+import application from '../application';
+
+const Notification = Mn.ItemView.extend({
+
+    template: require('./templates/notification'),
+
+    ui: {
+        notification: '.notification',
+    },
+
+    initialize(options) {
+        this.message = options.message;
+    },
+
+    onRender() {
+        this.ui.notification.show();
+        this.ui.notification.addClass('show');
+        setTimeout(function() {
+            $('.notification').hide();
+        }, 4500);
+    },
```

Comment:
Please avoid the use of jQuery's DOM effects method as it makes rendering parts complex to understand (i.e. is the transition effect in the CSS or in the JS code?). Prefer the use of dedicated classes or attributes that handles the effect in CSS (like `[aria-hidden=true] { display: none }`)